packages/containers/src/SidePanel/SidePanel.test.js

```diff
@@ -19,42 +22,214 @@ describe('SidePanel', () => {
 });
 
 describe('SidePanel.mapStateToProps', () => {
+	let state;
+	beforeEach(() => {
+		state = mock.state();
+		state.routing = {
+			locationBeforeTransitions: {
+				pathname: '/test',
+			},
+		};
+	});
 	it('should check for each action if one goes to the current route', () => {
-		const state = mock.state();
 		const props = mapStateToProps(state, {
-			location: { pathname: '/test' },
 			actionIds: ['menu:routerReplace'],
 		});
 		expect(props.actions[0].active).toBe(true);
 
 		const notactive = mapStateToProps(state, {
-			location: { pathname: '/different' },
-			actionIds: ['menu:routerReplace'],
+			actionIds: ['menu:routerPush'],
 		});
 		expect(notactive.actions[0].active).toBeUndefined();
 
+		state.routing.locationBeforeTransitions.pathname = '/push';
 		const push = mapStateToProps(state, {
-			location: { pathname: '/push' },
 			actionIds: ['menu:routerPush'],
 		});
 		expect(push.actions[0].active).toBe(true);
 
+		state.routing.locationBeforeTransitions.pathname = '/href';
 		const href = mapStateToProps(state, {
-			location: { pathname: '/href' },
 			actionIds: ['menu:href'],
 		});
 		expect(href.actions[0].active).toBe(true);
 	});
 
 	xit('should handle actionCreator with href', () => {
```

Comment:
why is it disabled ?

Reply:
It was already, the case was not covered in the code.